OpenRA.Game/Map/Map.cs

```diff
@@ -722,13 +722,16 @@ public void MakeDefaultPlayers()
 				Players.Add(p.Name, p);
 			}
 
-			Players.Add("Creeps", new PlayerReference
+			if (!Players.ContainsKey("Creeps"))
```

Comment:
I resolved it by creating a new `UpdateMultiPlayers` function and trigger that one when someone places a `mpspawn` actor.